tests/unit_tests/test_unit.py

```
@@ -27,17 +29,234 @@
+    output_file = tmp_path / "output.tsv"
+    ground_truth = pdb.db_peptides
+
+    loaded = pd.read_csv(output_file, sep="\t", index_col=0)
+    ground_truth["protein"] = ground_truth["protein"].apply(str)
+    loaded["protein"] = loaded["protein"].astype(str)
+
+    pd.testing.assert_frame_equal(ground_truth, loaded)
+
+
+@pytest.mark.parametrize(
+    "enzyme, specificity, expected, use_sort",
```

Ruff (0.15.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)